docs/ext/pydantic_autosummary/generate.py

```diff
@@ -0,0 +1,880 @@
+            # get_modules method, but only put in "public" modules that are in the
+            # __all__ list
+            #
+            # Otherwise, use get_modules method normally
+            if respect_module_all and "__all__" in dir(obj):
+                imported_modules, all_imported_modules = _get_members(
+                    doc, app, obj, {"module"}, imported=True
+                )
+                skip += all_imported_modules
+                imported_modules = [
+                    name + "." + modname for modname in imported_modules
+                ]
+                all_imported_modules = [
+                    name + "." + modname for modname in all_imported_modules
+                ]
+                public_members = getall(obj)
+            else:
+                imported_modules, all_imported_modules = [], []
+                public_members = None
+
+            modules, all_modules = _get_modules(
+                obj, skip=skip, name=name, public_members=public_members
+            )
+            ns["modules"] = imported_modules + modules
+            ns["all_modules"] = all_imported_modules + all_modules
+    elif doc.objtype == "class":
+        ns["members"] = dir(obj)
+        ns["inherited_members"] = set(dir(obj)) - set(obj.__dict__.keys())
+        ns["methods"], ns["all_methods"] = _get_members(
+            doc, app, obj, {"method"}, include_public={"__init__"}
+        )
+        ns["attributes"], ns["all_attributes"] = _get_members(
+            doc, app, obj, {"attribute", "property"}
+        )
+
+    if modname is None or qualname is None:
+        modname, qualname = _split_full_qualified_name(name)
+
+    if doc.objtype in ("method", "attribute", "property"):
+        ns["class"] = qualname.rsplit(".", 1)[0]
+
+    # ----------- pydantic_autosummary change
+    if doc.objtype == "pydantic_model":
+        set_pydantic_model_fields(ns, obj)
+    # ----------/ pydantic_autosummary change
+
+    shortname = qualname if doc.objtype == "class" else qualname.rsplit(".", 1)[-1]
+
+    ns["fullname"] = name
+    ns["module"] = modname
+    ns["objname"] = qualname
+    ns["name"] = shortname
+
+    ns["objtype"] = doc.objtype
+    ns["underline"] = len(name) * "="
+
+    return template.render(template_name or doc.objtype, ns)
+
+
+def _skip_member(app: Sphinx, obj: Any, name: str, objtype: str) -> bool:
+    try:
+        return app.emit_firstresult(
+            "autodoc-skip-member", objtype, name, obj, False, {}
+        )
+    except Exception as exc:
+        # ----------- pydantic_autosummary change
+        logger.warning(
+            __(
+                "pydantic_autosummary: failed to determine %r to be documented, "
+                "the following exception was raised:\n%s"
+            ),
+            name,
+            exc,
+            type="autosummary",
+        )
+        # ----------/ pydantic_autosummary change
```

Comment:
Just changed the logger to use `pydantic_autosummary: ` instead of `autosummary: `